docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -885,6 +1008,14 @@ NOTE: The size of a PQ on disk includes both unacknowledged events and previousl
       This means it grows gradually as individual events are added, but shrinks in large chunks each time a whole page of processed events is reclaimed (read more: <<garbage-collection, PQ disk garbage collection>>).
 |===
 
+[discrete]
+[[plugin-flow-rates]]
+===== Plugin flow rates
+
+Input plugin entry of pipeline in the API response includes `throughput` to provide visibility into the flow of events through the input plugin.
+
+Filter and output plugin entries of pipeline in the API response include `worker_utilization` metric, indicates how many percentage of work the workers spent when proceeding event processing through the plugin and `worker_cost_per_event` metric, indicates how much time the workers spent on each event while processing through the plugin.
```

Comment:
@yaauie as always I have a favor for you to review these important lines. As much as I think about `worker_cost_per_event`, I have only one candidate which similar `worker_time_per_event`. Let me please know if we can rename to better candidate. Thanks a lot!